### DISCUSSION TITLE
code_actions problems

### DISCUSSION THREAD

**Discussion by zephod77** [folke/noice.nvim#1022] [General]
I'm running nvim v0.11.0-dev-1351+gf9dd682621 and noice commit 7b1960c.

I have a key mapping defined for code actions like this:
```vim.keymap.set({'n','v'},'<C-k>', vim.lsp.buf.code_action,{})```
The first problem is that when I press <ctrl>-k, an empty box pops up and another 'Confirm' box pops up like this:
![image](https://github.com/user-attachments/assets/a87d995f-120c-40ad-978b-85849e0860fe)

So the first problem is that the code actions box is empty and the seconds is that the 'Confirm' overlays the box so I couldn't read it anyhow.
If I press <enter>, what is in the box becomes visible:
![image](https://github.com/user-attachments/assets/2b10a456-1418-405d-9323-43c4af84fe60)

but now pressing 1 doesn't do anything.

If I remove noice, then the code actions are visible:
![image](https://github.com/user-attachments/assets/40ad6a4f-7e5f-43bc-a31f-e3400e705b73)

Is this a known issue and is there a workaround?

**Comment by luukvbaal** [comment]
Related: #968, #1002; https://github.com/neovim/neovim/pull/31525 will address this.

**Reply by zephod77** [reply]
Ok, thanks. I will look for a fix in the new year. For now, I'll disable noice.
